lib/proto/xudrpc.swagger.json

```diff
@@ -21,7 +21,7 @@
         "operationId": "AddCurrency",
         "responses": {
           "200": {
-            "description": "",
+            "description": "A successful response.",
```

Comment:
Can you try regenerating these with the version of grpc-swagger I specified in a separate PR? I can find it for you again if you need.

Reply:
got it thanks :)